profile/prefs.js

```diff
@@ -74,3 +74,6 @@ user_pref("experiments.manifest.uri", "");
 user_pref("experiments.supported", false);
 user_pref("experiments.activeExperiment", false);
 user_pref("network.allow-experiments", false);
+
+// Marionette settings
+user_pref("marionette.log.level", "debug");
```

Comment:
how does this one looks like? does it spam a lot to the terminal?

Reply:
Sadly, even with this there are almost no logs :\ makes it really hard to debug Marionette which is already a document-less project